Bottom: 92b9ab4c364b52023f3a50d91d7174c55cdc0e6b
Top:    7b88c035c4befc5a9235246d026db4975ea6b368
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-01 17:11:02 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
